// Copyright 2023 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// If you change or add any fields in this file, update proto_visitors.h and
// potentially proto_enum_conversions.{h, cc}.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

// Autofill wallet data which stores the relevant card credential data.
message AutofillWalletCredentialSpecifics {
  // A server-generated unique ID for the instrument/card.
  optional string instrument_id = 1;

  // A 3 or 4 digit card verification code.
  optional string cvc = 2;

  // The timestamp of the last update. Used to determine data freshness.
  optional int64 last_updated_time_unix_epoch_millis = 3;
}
